interactive_engine/executor/store/groot/src/api/condition/operand.rs

```diff
@@ -0,0 +1,47 @@
+use crate::api::{property::Property, Edge, PropId, Vertex};
+
+#[derive(Debug, Clone, PartialEq)]
+pub enum Operand {
+    Const(Property),
+    PropId(PropId),
+    Label,
+    Id,
+}
+
+impl Operand {
+    pub(crate) fn get_const_property(&self) -> Option<&Property> {
+        match self {
+            Operand::Const(prop) => Some(prop),
+            _ => None,
+        }
+    }
+    pub(crate) fn extract_vertex_property<V: Vertex>(&self, vertex: &V) -> Option<Property> {
+        match self {
+            Operand::PropId(prop_id) => vertex.get_property(*prop_id),
+            Operand::Label => {
+                let label_id = vertex.get_label_id();
+                Some(Property::Long(label_id as i64))
```

Comment:
you means the real label_id value passed by ir, such as `Operand::Label Eq Oprand::Const(Object::Primitive(Primitives::Long(lable_id)))` ?.  Previously, I improved the `PartialEq` trait for `Property` [here](https://github.com/alibaba/GraphScope/commit/17585255871676dd043d86556590f2a207c16ee7), all number types could be compared with each other, if type miss match, if one of them is a floating type, both would be conveted to f64, other wise, both would be converted to i64. So, whatever the type of `label_id` ir passed, Primitives::i32 or i64, it can compare with  `Property::Long`.